### ISSUE TITLE
Softmax GAN

### ISSUE THREAD

**Issue by 0shimax** [arXivTimes/arXivTimes#280]
## 一言でいうと
GANの学習安定化のためにclassification lossをbinaryでなくてMulticlass cross entropy loss(softmax loss)にした。

### 論文リンク
https://arxiv.org/abs/1704.06191

### 著者/所属機関
Min Lin
